subsys/net/lib/lwm2m/Kconfig

```diff
@@ -385,6 +385,21 @@ config LWM2M_RW_JSON_SUPPORT
 	help
 	  Include support for writing JSON data
 
+
+config LWM2M_RW_SENML_JSON_SUPPORT
+	bool "support for SENML JSON data format"
+	depends on BASE64
+	help
+	  Include support for write / parse SENML JSON data
+
+config LWM2M_COMPOSITE_PATH_LIST_SIZE
+	int "Maximum # of composite read  and send operation URL paths"
+	default 6
+	help
+	  Define path list size for Composite Read and and send operation.
+	  Every entry in path list structure will increase stack usage by 13
```

Comment:
Better not to put the exact number here. Just one more thing which might drift out of sync.